app/views/registration_commands/_form.erb

```diff
@@ -48,28 +48,36 @@
   </div>
   <div class='form-group'>
     <label class='col-md-2 control-label' for='setup_insights'>
-      <% help = _('If selected, Insights client will be installed and registered on Red Hat family operating systems. It has no effect on other OS families that do not support it.') %>
+      <% help = _('If selected, Insights client will be installed and registered on Red Hat family operating systems. It has no effect on other OS families that do not support it. Default value is defined by `host_registration_insights` global parameter.') %>
       <%= _('Setup Insights') %>
       <a rel="popover" data-content="<%= help %>" data-trigger="focus" data-container="body" data-html="true" tabindex="-1">
         <span class="pficon pficon-info "></span>
       </a>
     </label>
     <div class='col-md-4'>
-      <% insights_opts = options_for_select([[_('Yes'), 'true'], [_('No'), 'false']], params[:setup_insights]) %>
-      <%= select_tag 'setup_insights', insights_opts, class: 'form-control', include_blank: '' %>
+      <%
+        host_param = @host_config_params['host_registration_insights'] ? _('Yes') : _('No')
+        default_opt = [(_("Default (%s)") % host_param), '']
+        insights_opts = options_for_select([default_opt, [_('Yes (override)'), 'true'], [_('No (override)'), 'false']], params[:setup_insights])
+      %>
+      <%= select_tag 'setup_insights', insights_opts, class: 'form-control' %>
     </div>
   </div>
   <div class='form-group'>
     <label class='col-md-2 control-label' for='setup_remote_execution'>
       <%= _('Remote Execution') %>
-      <% help = _('If selected, SSH keys will be installed on the registered host.') %>
+      <% help = _('If selected, SSH keys will be installed on the registered host. Default value is defined by `host_registration_remote_execution` global parameter.') %>
```

Comment:
I'd suggest the following wording

```
If set to "Yes", SSH keys will be installed on the registered host. The default value is based on the
`host_registration_remote_execution` parameter. It can be inherited e.g. from host group, operating system, organization.. When overidden, the selected value will be stored on host parameter level.
```